server/sonar-db-dao/src/main/java/org/sonar/db/rule/RuleDao.java

```diff
@@ -36,6 +36,14 @@
 
 public class RuleDao implements Dao {
 
+  public Optional<RuleDto> selectByKey(DbSession session, OrganizationDto organization, RuleKey key) {
+    return selectByKey(session, organization.getUuid(), key);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Remove this use of "selectByKey"; it is deprecated. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)